File: fizz_buzz.pl
# Fizz Buzz in a one line Perl script
# (see https://wiki.c2.com/?FizzBuzzTest )
# You can run this code at:
# https://www.tutorialspoint.com/execute_perl_online.php 
for (1..20) { print $_ % 3 ? "" : "fizz" ; print $_ % 5 ? "" : "buzz" ; print $_ % 3 && $_ % 5 ? $_ : "" ; print "\n" }

File: Fizz_Buzz.java
/*
 * Java FizzBuzz
 * You can run this code at:
 * https://www.tutorialspoint.com/compile_java_online.php 
 **/
public class Fizz_Buzz {
  static public void main(String[] args) {
    int i = 0;
    for (i = 1; i < 101; i++) {
      if (i % 3 == 0) {
        System.out.print("Fizz");
      }
      if (i % 5 == 0) {
        System.out.print("Buzz");
      }
      if (i % 3 != 0 && i % 5 != 0) {
        System.out.print(i);
      }
      System.out.println();
    }
  }
}
